packages/camera/camera_avfoundation/ios/camera_avfoundation/Sources/camera_avfoundation/FLTCam.m

```diff
@@ -1320,6 +1362,12 @@ - (void)setUpCaptureSessionForAudio {
   // Setup the audio output.
   _audioOutput = [[AVCaptureAudioDataOutput alloc] init];
 
+  upgradeAudioSessionCategory(AVAudioSessionCategoryPlayAndRecord,
+                              AVAudioSessionCategoryOptionDefaultToSpeaker |
+                                  AVAudioSessionCategoryOptionAllowBluetoothA2DP |
+                                  AVAudioSessionCategoryOptionAllowAirPlay,
+                              0);
```

Comment:
Do we only pass `0` here? Should we just remove this parameter?

Reply:
It looked like a good idea during development as there is this same function also in the player which uses that argument and it was easier to work on it in a single place and then copy it into the other plugin. One time this argument was actually used when I tried to use `AVAudioSessionCategoryOptionAllowBluetooth` it looked like a good idea to remove that option when the camera was no longer used.